### PR TITLE
Some Basic UI Love

### PR DESCRIPTION
**Pull request by tpreusse**
screen shot 2017-09-03 at 22 42 41
screen shot 2017-09-03 at 22 43 16

Changes
- added a header
- new repo nav
- fixed sidebars
- fixed document empty state
- rm italic button (we have no italic font styles)

### CODE REVIEW

File: components/Frame/Header.js [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+  string.split('@')[0]
+    .replace(/\s*\.\s*/, ' ')
+    .split(' ')
+    .map(part => part[0].toUpperCase() + part.slice(1))
+    .join(' ')
+)
+
+const styles = {
+  bar: css({
+    zIndex: 20, // goes over footer
```
Comment (uxengine) [inline components/Frame/Header.js]: Optional:
We might wanna move these to one central zIndex file at some point.

File: components/Frame/LoadingBar.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
+import React, { Component } from 'react'
+import Router from 'next/router'
+// import track from '../../lib/piwik'
```
Comment (uxengine) [inline components/Frame/LoadingBar.js]: Remove?

File: components/Frame/LoadingBar.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+import { css } from 'glamor'
+import { colors } from '@project-r/styleguide'
+
+const styles = {
+  loadingBar: css({
+    position: 'fixed',
+    zIndex: 30,
```
Comment (uxengine) [inline components/Frame/LoadingBar.js]: Optional:
We might wanna move these to one central zIndex file at some point.

File: components/Frame/LoadingBar.js [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+    Router.onRouteChangeStart = () => {
+      clearTimeout(this.timeout)
+      this.setState({ loading: true, progress: 0.02 })
+    }
+    Router.onRouteChangeComplete = url => {
+      clearTimeout(this.timeout)
+      this.setState({ loading: false })
+
+      // update url manually, seems necessary after client navigation
+      // track(['setCustomUrl', window.location.href])
```
Comment (uxengine) [inline components/Frame/LoadingBar.js]: Remove?

**Comment by tpreusse** [comment]
Thansk Dani, will add Piwik and zIndex in a separate PR

**Comment by tpreusse** [comment]
Adressed in: https://github.com/orbiting/haku-frontend/commit/fc49dc084c6d6670f9a70f19a92f847b1d83d567 and https://github.com/orbiting/haku-frontend/commit/ca36a5f77722b2acd8af33a93d6ac947ed06c5f3

**Pull request merged**